packages/foreman/foreman-installer/foreman-installer.spec

```diff
@@ -1,17 +1,15 @@
-# set and uncomment all three to set alpha tag
-#global alphatag RC1
-#global dotalphatag .%{alphatag}
-#global dashalphatag -%{alphatag}
+%global prerelease develop
+%global release 1
```

Comment:
One thing to consider here: Previously we had `1.20.0-0.develop` followed by then `1.20.0-0.1RC1`. Because `d < 1` it allowed upgrades. Since we're now including releases we could end up with `1.20.0-0.5.develop` which means we'll also have `1.20.0-0.5RC1`. This will work but because release is more important than prerelease in ordering I'd like it to come first in the file (so swap these 2 lines). Otherwise you might be tempted to reset the release to 1 when bumping to RC1. Small and subtle, but I have made this mistake in the past. Now we have automation that should catch it, but it's better to avoid it in the first place.

Reply:
Sorry, I don't understand what the problem is that we may encounter by this change. When we do a release, we'll have 1.20.0-0.1.RC1 which is what we had before and want as I understand things.

Reply:
It was a long way of saying that release is more significant than preprelease in determining the order. That's why I'd like the release macro to be defined before prerelease. Currently it's the other way around. This is no difference on a technical level but may help prevent human errors in the future.

Reply:
Heh, I can do that if you want. A bit ironic that prerelease will come after release ;)